shared/devices/device-page/dumb.native.js

```diff
@@ -1,92 +0,0 @@
-// @flow
```

Comment:
don't think we actually need a mobile specific version of most of these dumb sheets